ClassLibrary/ClassLibrary/Library.cs

```diff
@@ -1,13 +1,39 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace ClassLibrary
 {
     public class Library
     {
         public List<Book> Books= new List<Book>();
-        public List<CardReader> CardReaders = new List<CardReader>();
-        public List<Reader> Readers = new List<Reader>();
+
+        public List<Author> Authors = new List<Author>();
+
+        //букк пока не смотри
```

Comment:
please remove `AddBook`, then we'll merge this PR and `AddBook` implementation will be in a new one.